pkg/bbgo/order_executor_general.go

```diff
@@ -262,7 +297,7 @@ func (e *GeneralOrderExecutor) reduceQuantityAndSubmitOrder(ctx context.Context,
 	return nil, multierr.Append(ErrExceededSubmitOrderRetryLimit, err)
 }
 
-func (e *GeneralOrderExecutor) OpenPosition(ctx context.Context, options OpenPositionOptions) (types.OrderSlice, error) {
+func (e *GeneralOrderExecutor) OpenPositionPrepare(ctx context.Context, options *OpenPositionOptions) (*types.SubmitOrder, error) {
```

Comment:
make this private?

Reply:
this is called inside drift.  
reuse the login in `GeneralOrderExecutor` to feed the `*submitOrder` to `FastSubmitOrders` or `BatchPlaceOrder`